src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -21,7 +21,11 @@
     aria-label="API Reference"
   >
     <template #pre-title="{ closeNav }" v-if="isWideFormat">
-      <button class="sidenav-toggle" @click.prevent="handleSidenavToggle(closeNav)">
+      <button
+        aria-label="Toggle documentation navigator"
```

Comment:
Since this button is for opening the navigator only, should we change the wording here to "Open documentation navigator" to make match what the aria label for "Close documentation navigator"?

Reply:
True, good point, Hanqing, I changed it :) thanks! 👏